poc/vdaf_prio3.py

```diff
@@ -483,6 +510,51 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+class Prio3SumVecWithMultiproof(Prio3SumVec):
```

Comment:
We want to make it clear that this is not (yet) on the standards track. We could prefix the class name with `Test`, or maybe just add a comment clarifying what this is intended for.

Reply:
OK. I will add a comment.